modules/katello_content_view.py

```diff
@@ -183,12 +184,7 @@ def main():
     )
     ng = NailGun(server, entities, module)
 
-    # Lets make an connection to the server with username and password
-    try:
-        org = entities.Organization(server)
-        org.search()
-    except Exception as e:
-        module.fail_json(msg="Failed to connect to Foreman server: %s " % e)
+#    ping_server(module)
```

Comment:
for `ping_server` you'd need to also call `create_server` from `ansible_nailgun_cement` like this:

```python
    try:
        create_server(server_url, (username, password), verify_ssl)
    except Exception as e:
        module.fail_json(msg='Failed to connect to Foreman server: %s ' % e)

    ping_server(module)
```

but as the module currently is not migrated to cement, I see no benefit in doing so for now